db/seeds.rb

```diff
@@ -51,16 +55,18 @@ def seed_samples(project, sample_count)
       { name: "#{project.namespace.parent.name}/#{project.name} Sample #{i}",
         description: "This is a description for sample #{project.namespace.parent.name}/#{project.name} Sample #{i}." }
     ).execute
-    seed_attachments(sample) if i < 10
+    seed_attachments(sample) if @sample_attachment_count < @sample_attachment_limit
```

Comment:
The way you currently have it setup, it will add up to 20 attachments to one sample for the first group -> project and the rest of the samples in this project and all other projects will have no attachments. What I think we need to do is update the `sequencing_files` method to limit the number of files returned instead. This way we get attachments for all the samples if the the user provided the env variable above.

In the sequencing_files method you should be able to add a `.first(@sample_attachment_count)` to the select

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/310/commits/ec1705f3c953272d7404d7efb995d35d49d31cf3